### DISCUSSION TITLE
Pair of Glyphs

### DISCUSSION THREAD

**Discussion by EddCase** [moudey/Shell#484] [Q&A] [answered]
Back in the patch notes for Version 1.9 9/27/2023 its mentioned that 

> New option to add a pair of glyphs

can anyone confirm what the syntax for doing that is, I can add glyphs and change their colour e.g.
`modify(find = "Drop As*" pos="top" image=[\uE13C,#4CFF00] sep=after)`
but I cannot figure out how to add a second.

Many Thanks

**Comment by ghost** [comment, marked as answer]
```js
item(title="Something" image=[[\uE019],[\uE01A]]) or image=[[\uE00B, #ffff00], [\uE70F, #ff00ff], 'Segoe MDL2 Assets']
```

**Reply by EddCase** [reply]
Brilliant, thank you.

**Reply by NIKOLENKO-TE** [reply]
> ```js
> item(title="Something" image=[[\uE019],[\uE01A]]) or image=[[\uE00B, #ffff00], [\uE70F, #ff00ff], 'Segoe MDL2 Assets']
> ```

How to pair image.fluent?
